web/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -3,6 +3,17 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
 
+
+    <bean class="org.springframework.context.support.PropertySourcesPlaceholderConfigurer">
```

Comment:
use more convenient way to declare config properties: `context:property-placeholder`